docker/dockerfiles/be/be_entrypoint.sh

```diff
@@ -132,7 +132,7 @@ while true; do
       tail -n $nol $STARROCKS_HOME/log/be.out
   fi
 
-  if [[ "$COREDUMP_ENABLED" != "true" ]]; then
+  if [[ "$COREDUMP_ENABLED" != "true" || $ret -eq 0 ]]; then
```

Comment:
> 128+n | Fatal error signal “n”

if coredump enabled, only catch SIGABRT (6), SIGSEGV(11) and repeat launching be process, should still exit for rest error codes.

Reply:
So do you mean:
```
if [[ "$COREDUMP_ENABLED" != "true" || $ret != 134 || $ret != 139 ]]; then
```

Reply:
Updated